### ISSUE TITLE
Scene group or netsed navigation

### ISSUE THREAD

**Issue by mahramane** [Tlaster/PreCompose#53]
In jetpack compose we have `navigation`  in NavHost
Witch make a group of screens with a route address
Like the following code:

```
@Composable
fun CuteDogPicturesApp() {
	val navController = rememberNavController()
	NavHost(navController, startDestination = Screen.Feed) {
		composable(route = Screen.Feed) {
			FeedScreen(
				showAdoptionPage = { dogId ->
					navController.navigate(Screen.Adopt.createRoute(dogId))
				}
			)
		}
		navigation(route = Screen.Dog, startDestination = DogScreen.Adopt) {
		    composable(route = DogScreen.Adopt) { backStackEntry ->
			val dogId = backStackEntry.arguments?.getString("dogId")
			requireNotNull(dogId) { "dogId parameter wasn't found. Please make sure it's set!" }
			AdoptionScreen(
				dogId,
				navigateUp = { navController.popBackStack(DogScreen.Adopt, inclusive = true) }
			)
		    }
		    composable(route = DogScreen.ContactDetails) { backStackEntry ->
			val dogId = backStackEntry.arguments?.getString("dogId")
			requireNotNull(dogId) { "dogId parameter wasn't found. Please make sure it's set!" }
			AdoptionContactDetailsScreen(dogId)
		    }
		}
		// imagine 30 more of these in here!
	}
}
```

Can i create it with precompose?

**Comment by mahramane** [comment]
this is very important. 
Please add it soon so we can use it for our project.

**Comment by Tlaster** [comment]
We will have a new release after I fix #56
